### PR TITLE
Bug 1245608 - implement slideshow iframe, r=Standard8

### PR DESCRIPTION
**Pull request by dmose**

### CODE REVIEW

File: shared/js/utils.js [hunk 1]
```diff
@@ -6,7 +6,19 @@
 
 var loop = loop || {};
 loop.shared = loop.shared || {};
-var inChrome = typeof Components != "undefined" && "utils" in Components;
+var inChrome = typeof Components != "undefined" &&
+  "utils" in Components;
+
+// The slideshow is special, and currently loads with chrome privs, but
+// needs to use this module like the rest of the content does.  Once we make
+// it load remotely, and this can go away.
```
Comment (dmose) [inline shared/js/utils.js]: s/, and/,/

File: add-on/chrome/bootstrap.js [hunk 2]
```diff
@@ -117,6 +139,13 @@ var WindowListener = {
           });
         }
 
+        if (this.isSlideshowOpen) {
+          // XXX or return nothing?
+          return new Promise(resolve => {
+            resolve();
```
Comment (Standard8) [inline add-on/chrome/bootstrap.js]: Change to `return Promise.resolve();`

File: add-on/chrome/modules/MozLoopAPI.jsm [hunk 3]
```diff
@@ -1020,7 +1017,7 @@ const LoopAPIInternal = {
 
     Cu.import("resource://gre/modules/RemotePageManager.jsm");
 
-    gPageListeners = [new RemotePages("about:looppanel"), new RemotePages("about:loopconversation")];
+    gPageListeners = [new RemotePages("about:looppanel"), new RemotePages("about:loopconversation"), new RemotePages("chrome://loop/content/panels/slideshow.html")];
```
Comment (Standard8) [inline add-on/chrome/modules/MozLoopAPI.jsm]: Please add comment about why slideshow is here. Also consider wrapping lines.

**Comment by Standard8** [comment]
General: grep repo and check for `latestFTUVersion` bumps required (except `prefs.js`)

File: add-on/chrome/modules/MozLoopService.jsm [hunk 4]
```diff
@@ -1880,20 +1893,64 @@ this.MozLoopService = {
...
+    let appContent = xulDoc.getElementById("appcontent");
+    let tabBrowser = xulDoc.getElementById("content");
+    appContent.insertBefore(box, tabBrowser);
+
+    var xulBrowser = xulDoc.createElementNS(kNSXUL, "browser");
+    xulBrowser.setAttribute("id", "loop-slideshow-browser");
+
+    xulBrowser.setAttribute("flex", "1");
+    xulBrowser.setAttribute("type", "content");
+    box.appendChild(xulBrowser);
```
Comment (Standard8) [inline add-on/chrome/modules/MozLoopService.jsm:1954]: nit: suggest grouping the setAttributes together with whitespace either side.

File: add-on/chrome/modules/MozLoopService.jsm [hunk 5]
```diff
@@ -1880,20 +1893,64 @@ this.MozLoopService = {
...
+
+    xulWin.LoopUI.isSlideshowOpen = true;
```
Comment (Standard8) [inline add-on/chrome/modules/MozLoopService.jsm:1958]: Add a comment

**Pull request closed without merging**